installing/installing_with_agent_based_installer/prepare-pxe-assets-agent.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: ASSEMBLY
+[id="prepare-pxe-assets-agent"]
+= Preparing PXE assets for {product-title}
+include::_attributes/common-attributes.adoc[]
+:context: prepare-pxe-assets-agent
+
+toc::[]
+
+Use the following procedures to create the assets needed to PXE boot an {product-title} cluster using the Agent-based Installer.
+
+The procedures create assets to deploy a single-node {product-title} installation. You can use these procedures as a basis and modify according to your requirements.
+
+[id="prerequisites_prepare-pxe-assets-agent"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+// Downloading the Agent-based Installer
+include::modules/installing-ocp-agent-download.adoc[leveloffset=+1]
+
+// Creating the preferred configuration inputs
+include::modules/installing-ocp-agent-inputs.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* See xref:../../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#modifying-install-config-for-dual-stack-network_ipi-install-installation-workflow[Deploying with dual-stack networking].
+* See xref:../../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#configuring-the-install-config-file_ipi-install-installation-workflow[Configuring the install-config yaml file].
+* See xref:../../installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc#installation-three-node-cluster_installing-restricted-networks-bare-metal[Configuring a three-node cluster] to deploy three-node clusters in bare metal environments.
+* See xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#root-device-hints_preparing-to-install-with-agent-based-installer[About root device hints].
+* See link:https://nmstate.io/examples.html[NMState state examples].
```

Comment:
```suggestion
* link:https://nmstate.io/examples.html[NMState state examples].
```
The latest guidance I have seen is that you only need "See" if you have other text needed after the link. As little additional text outside the link text as possible is the general preference.

Reply:
Sounds good, thanks. I copied this content over from another install doc but did a lazy job of editing this section, that's on me.